### ISSUE TITLE
Should Shuup have refund orders?

### ISSUE THREAD

**Issue by tulimaki** [shuup/shuup#1177]
Now Shuup has refund order lines and stocks is considered while doing refunds.

* How creating refunds effect on payments? 
* What should sales reports say for January if the shop has 1 $500 order on January which is refunded on February is it ok that sales report for January is all of the sudden $0?
* Should there be cutoff date until refunds can be made for the order? Like tax changes or provisions paid to sales?
* Is it correct that Shuup modifies the order by creating new lines while refunding? I think there would be place for refund-orders in the world of eCommerce.

**Comment by JoaRiski** [comment]
My opinion regarding the second point is that refunding an order from January in February should not modify January's sales report. This because it would invalidate all reports that have already been possibly handed out, and those really should be immutable after the month has ended. Maybe the behavior should be made configurable?

**Comment by tulimaki** [comment]
Yeah it seems there is a need for both behaviors. For now we don't have need to change this, but surely something we want to address this before reaching 2.0.

**Comment by finmaster19** [comment]
This is exactly what some shuup merchants have addressed. I will talk about this and possible implementations with others and we'll see if there is something that very likely should be done about refunding orders.
